DEV-GUIDE.md

```diff
@@ -195,6 +196,41 @@ undesirable as Golang may not erase out of date artifacts from the
 cache. In such a situation, erase the contents of `$GOPATH/pkg/` manually
 before rebuilding the code.
 
+### Building the code with debugger information
+
+If you'd like to use a debugger such as delve to debug istio containers, run
+
+```shell
+make BUILD=debug
```

Comment:
I'm +1 on making debug the default - we also build with proxy-debug variant. 

But we do need to add a 'release' target, which downloads the optimized proxy and builds go with the optimizations.

Reply:
@costinm not sure about the specifics of golang, however, with atleast C the debuginfo is helpful even when debugging optimized code - especially for system tools to auto-report coredumps.  debuginfo has no cost other than size in GCC.  Do you know if it is possible to turn on debuginfo in goland while also optimizing?

Cheers
-steve